exhibitions/src/main/res/layout/fragment_exhibition_details.xml

```diff
@@ -0,0 +1,186 @@
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:alpha="0"
+                    android:ellipsize="end"
+                    android:fontFamily="@font/ideal_sans_medium"
+                    android:lines="1"
+                    android:textColor="@color/white"
+                    android:textSize="21sp" />
+
+            </android.support.v7.widget.Toolbar>
+
+
+        </android.support.design.widget.CollapsingToolbarLayout>
+
+    </android.support.design.widget.AppBarLayout>
+
+    <android.support.v4.widget.NestedScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/greyText"
+        android:fillViewport="true"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <android.support.constraint.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/greyText">
+
+            <TextView
+                android:id="@+id/metaData"
+                style="@style/MetaDataFifteenWhite"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/marginDouble"
+                android:layout_marginTop="13dp"
+                android:layout_marginEnd="@dimen/marginDouble"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                tools:text="June 30, 1:00pm" />
+
+            <android.support.constraint.Guideline
+                android:id="@+id/centerGuide"
+                android:layout_width="@dimen/marginDouble"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                app:layout_constraintGuide_percent=".5" />
+
+            <Button
+                android:id="@+id/showOnMap"
+                android:layout_width="0dp"
+                android:layout_height="50dp"
+                android:layout_marginStart="@dimen/marginDouble"
+                android:layout_marginTop="21dp"
+                android:layout_marginEnd="@dimen/marginStandard"
+                android:background="@color/sea"
+                android:fontFamily="@font/ideal_sans_medium"
+                android:textAllCaps="false"
+                android:textColor="@color/white"
+                android:textSize="14sp"
+                app:layout_constraintEnd_toStartOf="@id/centerGuide"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/metaData"
+                tools:text="Show on Map"
+
+                />
+
+            <Button
+                android:id="@+id/buyTickets"
+                android:layout_width="0dp"
+                android:layout_height="50dp"
```

Comment:
fixed button height is not ideal. can we add sufficient padding so its emulated, but can still properly resize for accessibility?